packages/lit-dev-content/samples/examples/react-basics/index.tsx

```diff
@@ -0,0 +1,7 @@
+import {React, ReactDOM} from './react.js';
```

Comment:
I think this playground has two important files (the component and the usage of it), and three others that aren't so important and a bit distracting.

Can we narrow this down by combining `app.tsx` and `index.tsx`? Would we do the same for the other examples? wdyt @sorvell ?

Reply:
sounds good to me!

Reply:
I've combined app.tsx and index.tsx on all examples.
Done!